### PR TITLE
Added latest attending.io link

### PR DESCRIPTION
**Pull request by iteles**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,13 +28,14 @@
...
-          <span class="glyphicon glyphicon-time text-muted"></span> 6:30pm Thursday 5th February<br/>
+          <span class="glyphicon glyphicon-time text-muted"></span> 6:30pm Thursday 5th March<br/>
+          <span class="glyphicon glyphicon-time text-muted"></span> Bowling starts at 7:00pm sharp!<br/>
           <span class="glyphicon glyphicon-map-marker text-muted"></span> Palace Superbowl, Elephant &amp; Castle, SE1 6TE
           </b>
         </p>
         <h3>
           <span class="glyphicon glyphicon-arrow-right text-muted"></span>
-          <a href="http://attending.io/events/bowlingjs">Register here</a>
+          <a href="http://attending.io/events/bowlingjs-bd686207-34e9-43df-a597-f09285cb2678">Register here</a>
```
Comment (nelsonic) [inline index.html]: why does @attending use such silly urls! :-1:

File: index.html [hunk 2]
```diff
@@ -57,7 +58,7 @@ <h2 class="text-center">Who can come?</h2>
         <span class="glyphicon glyphicon-star text-yellow"></span>
       </p>
       <h2 class="text-center">How does it work?</h2>
-      <p>Grab a ticket when the meetup is announced. We'll book lanes at the alley and all you need to do is turn up and pay for your bowling. If you don't get a ticket we can't guarantee there'll be space on the lane.</p>
+      <p>Grab a ticket when the meetup is announced. We'll book lanes at the alley and all you need to do is turn up and pay for your bowling. Ifyou don't get a ticket or if you arrive after 7pm (when we start the bowling itself) we can't guarantee there'll be space on the lane.</p>
```
Comment (nelsonic) [inline index.html]: I can't tell if the **Ifyou** has a space between the two words or not... @iteles

**Comment by iteles** [comment]
@nelsonic Done!

**Comment by nelsonic** [comment]
*merged*.

**Pull request merged**